samples/sampler/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@enact/agate": "../../",
     "@enact/core": "^3.2.5",
```

Comment:
Do you have any reason why @enact/core still use 3.2.5? Other enact libraries uses 3.3.0-alpha.8. In my opinion, all enact libraries should use the same version.

Reply:
I will updated it.